.gitignore

```diff
@@ -10,7 +10,6 @@ deps/deps.jl
 *~
 
 # Documenter.jl and MkDocs
-docs/Manifest.toml

```

Comment:
@ali-ramadhan should `docs/Manifest.toml` be in the `.gitignore` or was I right to take it ou?